docs/framework/tools/regasm-exe-assembly-registration-tool.md

```diff
@@ -55,7 +55,7 @@ When you register an assembly for use by COM, Regasm.exe adds entries to the reg
 
 For example, consider a scenario where you register the managed component, myComp.dll, version 1.0.0.0 for use by COM. Later, you register myComp.dll, version 2.0.0.0. You determine that all COM client applications on the computer are using myComp.dll version 2.0.0.0 and you decide to unregister myComponent.dll version 1.0.0.0. This registry scheme allows you to unregister myComp.dll version 1.0.0.0 because only the version 1.0.0.0 subkey is removed.
 
-After registering an assembly using Regasm.exe, you can install it in the [global assembly cache](../app-domains/gac.md) so that it can be activated from any COM client. If the assembly is only going to be activated by a single application, you can place it in that application's directory.
+After registering an assembly using Regasm.exe, you can install it in the [global assembly cache](../app-domains/gac.md) so that it can be activated from any COM client. If the assembly is only going to be activated by a single application, you can place it in that application's directory. Using the **/codebase** option is an alternative to using the global assembly cache; however, the location of the assembly during registration is recorded globally and activation will failure if the assembly is moved. The **/codebase** option will load the assembly in a LoadFrom context that has additional considerations documented in <xref:System.Reflection.Assembly.LoadFrom%2A?displayProperty=nameWithType>.
```

Comment:
Hmm, probably not. The breaking change notification is about serialization that is not directly related to interop.

Reply:
Perhaps I should add that in the `LoadFrom` documentation instead?

Reply:
It would be most appropriate for the binary formatter. Binary formatter is the affected entrypoint and the component responsible for this behavior. The binary formatter has so much fine print that adding this one will just get lost.

I do not think any action is needed.